### ISSUE TITLE
Devices could listen to interface org.freedesktop.DBus.Properties

### ISSUE THREAD

**Issue by yannickadam** [WaeCo/node-bluez#11]
Hi there,

I believe that interface org.freedesktop.DBus.Properties is only registered for characteristics.
However, I think the library is missing some interesting feedback from bluez like: 

Connection status
```
signal time=1564601858.473981 sender=:1.9 -> destination=(null destination) serial=1094 path=/org/bluez/hci0/dev_DE_E3_F9_4C_B5_15; interface=org.freedesktop.DBus.Properties; member=PropertiesChanged
   string "org.bluez.Device1"
   array [
      dict entry(
         string "Connected"
         variant             boolean true
      )
   ]
   array [
   ]
```

Services resolved
```
signal time=1564601859.631083 sender=:1.9 -> destination=(null destination) serial=1145 path=/org/bluez/hci0/dev_DE_E3_F9_4C_B5_15; interface=org.freedesktop.DBus.Properties; member=PropertiesChanged
   string "org.bluez.Device1"
   array [
      dict entry(
         string "UUIDs"
         variant             array [
               string "00001800-0000-1000-8000-00805f9b34fb"
               string "00001801-0000-1000-8000-00805f9b34fb"
               string "00001805-0000-1000-8000-00805f9b34fb"
               string "0000180a-0000-1000-8000-00805f9b34fb"
               string "0000180d-0000-1000-8000-00805f9b34fb"
               string "0000180f-0000-1000-8000-00805f9b34fb"
               string "00001814-0000-1000-8000-00805f9b34fb"
               string "00001816-0000-1000-8000-00805f9b34fb"
               string "8e400001-f315-4f60-9fb8-838830daea50"
               string "fce61000-7802-4392-b6b7-96b18deaad22"
               string "fce62000-7802-4392-b6b7-96b18deaad22"
            ]
      )
      dict entry(
         string "ServicesResolved"
         variant             boolean true
      )
   ]
   array [
   ]
```

The device could then emit a "Connect" or "ServicesResolved" notification.
This would make it much easier than looping every 100ms and checking the ServicesResolved property.

Would you like a PR for this?

**Comment by WaeCo** [comment]
I am on the way of swapping the underlying [dbus](https://github.com/Shouqun/node-dbus) library for [dbus-next](https://github.com/dbusjs/node-dbus-next) which provides more features like promises.
This change will also expose the `PropertiesChanged` signal so you do not have to poll for it.

**Comment by WaeCo** [comment]
Implemented in next version v0.4.0
